holidayextras/holidayextras.github.com
# New look!

The keep with Holiday Extras new employer brand colours, the hungry geek blog has had a bit of a redesign.

__Extras__
- Changed the speakers page to be less complex
- Removed people who no longer work for HX from the team page
- Remove speaker events
- Added conference logos to speaker pages.

__To Test__
- Run the blog, check a few pages and ensure the colours work well

<img width="800" alt="screen shot 2015-09-25 at 14 25 33" src="https://cloud.githubusercontent.com/assets/778942/10101554/5032831e-6391-11e5-8726-32b6da51aa85.png">
